docs/tutorials/common-errors.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Debugging Common Errors"
+sidebarTitle: "Common Errors"
+icon: "brain"
+description: "Here are our solutions to common errors. Can't find what you're looking for? Share it with the [Mage Community](https://join.slack.com/t/mageai/shared_invite/zt-1adn34w4m-t~TcnPTlo3~5~d_0raOp6A) or open a PR to get started!"
+---
+## Intro
+
+
+### Changing columns
+
+Sometimes, we can't control what data upstream of our pipelines contains. 😔
```

Comment:
It's actually a little bit hard to understand what the issue is by just reading the doc.
It'll be helpful to give more context:
* Show what the error is.
* What block and database does the error and solution apply to?
The title "changing columns" is bit confusing as well.

Reply:
Also, for the errors, if you think we can solve it within Mage, we should do that instead of letting user find it in the doc.

Reply:
let me know what you think!